subsys/net/lib/coap/coap_link_format.c

```diff
@@ -535,6 +519,13 @@ int coap_well_known_core_get(struct coap_resource *resource,
 		if (r < 0) {
 			goto end;
 		}
+
+		if ((resource + 1) && (resource + 1)->path) {
+			r = append_to_coap_pkt(response, ",", 1, &remaining, &offset, ctx.current);
```

Comment:
nit, this line is a bit lengthy, let's break it to fit into char limit:
```suggestion
			r = append_to_coap_pkt(response, ",", 1, &remaining, 
					       &offset, ctx.current);
```

Reply:
Done Thanks